packages/plugins/src/foundry/decode.ts

```diff
@@ -451,11 +390,7 @@ export const makeFunctionCallDecodedAction = (
     const functionName = decoded ? decoded.functionName : 'call'
     const variables = decoded
       ? decoded.variables
-      : [
-          data.length > 1000
-            ? `Very large calldata. View it in Foundry's dry run file: ${dryRunPath}`
-            : data,
-        ]
+      : [data.length > 1000 ? `Calldata is too large to display.` : data]
```

Comment:
We don't execute a dry run anymore, so there's no dry run path to include here

Reply:
I'd be ok with removing this and just displaying the full calldata no matter what. Since we're only ever going to display the call data if we aren't able to find the artifact for the contract, I don't think it'll be very common for anyone to hit this anyway.

Reply:
I'll leave it as is to speed up this PR, but feel free to add that to the [Improve Preview ticket](https://linear.app/chugsplash/issue/CHU-324/improve-preview)